### DISCUSSION TITLE
GPS / Geofencing support?

### DISCUSSION THREAD

**Discussion by FileSubmit** [kimai/kimai#3913] [Questions & Answers]
Hi

**Comment by ghost** [comment]
Hi. I would also be interested in this. It would be nice if someone from the developers could answer.

**Comment by kevinpapst** [comment]
No. 

And I doubt that Geofencing would reliably work, considering that Kimai runs in the browser.

Tracking GPS positions is another topic.
Won't make it into Kimai core, maybe as paid feature of a plugin.
